cocos/core/data/object.ts

```diff
@@ -343,7 +343,8 @@ class CCObject implements EditorExtendableObject {
      *       _destruct: function () {
      *           for (var key in this) {
      *               if (this.hasOwnProperty(key)) {
-     *                   switch (typeof this[key]) {
+     *                   switch (typeo
+     * f this[key]) {
```

Comment:
why change this into two lines

Reply:
Good catch, i didn't modify it. May be the VSC modified it. I will fix it.

Reply:
fixed